packages/components/atoms/f-button/stories/Button.stories.js

```diff
@@ -20,9 +20,12 @@ export const ButtonComponent = () => ({
         },
         isFullWidth: {
             default: boolean('isFullWidth', false)
+        },
+        href: {
```

Comment:
Not sure the best way to test this in storybook, but this should just be applied as a standard attribute (may be worth looking up if it's possible to link attributes up to knobs or not).